client/tracking_status.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // GetTrackingUpdate requests the tracking status of a shipment.
-func (c *Client) GetTrackingUpdate(carrier, trackingNumber string) (*models.TrackingStatus, error) {
+func (c *Client) GetTrackingUpdate(carrier, trackingNumber string, shippoSubAccountID string) (*models.TrackingStatus, error) {
```

Comment:
i don't think we'll need subaccount here, there's a carrier but it's `usps` not the carrier id: https://docs.goshippo.com/shippoapi/public-api/#operation/GetTrack